src/Validation/Validator.php

```diff
@@ -2184,6 +2184,11 @@ public function integer(string $field, ?string $message = null, Closure|string|n
      */
     public function array(string $field, ?string $message = null, Closure|string|null $when = null)
     {
+        $message ??= $this->getValidationMessage(
+            'The provided value must be an array',
+            __d('cake', 'The provided value must be an array')
```

Comment:
Why not do the translation too inside the helper method instead of repeating the string here?

Reply:
Because the translation string needs to be found when extracting I18n translation strings.

https://book.cakephp.org/4/en/console-commands/i18n.html#generating-pot-files

As much as I remember we cannot use variables on this method because the strings won't be found that way.

May be I'm not up to date on that, though.